### DISCUSSION TITLE
keybinding cycle alternatives Swedish keyboard

### DISCUSSION THREAD

**Discussion by johanperjulius1** [community/community#7212] [Copilot Conversations] [answered]
Hi! The keybinding for cycling alternatives does not work out of the box for my Swedish keyboard. alt + [ on a Swedish keyboard is alt + shift + 8. How can I change the keybindings? 

Kind regards

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Please check the documentation about configuring Copilot here - https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/configuring.md

**Reply by mlangiu** [reply]
broken

**Comment by k3b** [comment]
see https://github.com/orgs/github-community/discussions/6993

**Reply by mlangiu** [reply]
broken
